presto-docs/src/main/sphinx/connector/raptorx.rst

```diff
@@ -0,0 +1,206 @@
+=======
+RaptorX
+=======
+.. contents::
+  :local:
+  :backlinks: none
+  :depth: 1
+
+Overview
+--------
+
+RaptorX is an internal project name aiming to boost query latency significantly beyond what vanilla Presto is capable of. Hierarchical cache work, is the key building block for RaptorX. With the support of the cache, we are able to boost query performance by 10X. This new architecture can beat performance oriented connectors like Raptor with the added benefit of continuing to work with disaggregated storage.
```

Comment:
internal to whom/where? Maybe just "RaptorX aims to boost..."
No comma after "cache work"